lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -1661,6 +1661,20 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
       return b.create<arith::MulIOp>(loc, lhs, rhs);
     }
   }
+  if (auto gtTensor = dyn_cast<AtenGtTensorOp>(op)) {
+    AtenGtTensorOp::Adaptor adaptor(operands);
+    Type dtype = payloadArgs[0].getType();
+
+    // TODO: Currently, the dominating dtype is the lhs type whereas the
+    // bitwidth of lhs and rhs should be compared to get the resulting dtype.
```

Comment:
Here you have to worry not just about bit-width, but also about whether a type is a float or int. At the moment this implementation will not work correctly if the first input tensor is an int tensor and the second is a float because the float will get truncated to an int.

Reply:
I see, I will work on the type-promotion specific to this later. But thanks for your inputs.